.github/workflows/cd_prepare_release.yml

```diff
@@ -4,6 +4,23 @@ on:
     branches:
     - 'release/**'
 jobs:
+  generate_matrix:
+    runs-on: ubuntu-18.04
+    container:
+      image: docker://alpine/git:v2.24.3
+    outputs:
+      matrix: ${{ steps.read.outputs.matrix }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - id: read
+        run: |
+          json="$(cat images_matrix.json)"
```

Comment:
Could we extract that in a dedicated script, in a. .ci folder for example? Since we reuse the same logic for many workflows.

Reply:
Maybe we could even create a dedicated private action to do all this stuff. But the scope of this is bigger than this PR.

Reply:
I'm not sure if I should have moved `images_matrix.json` as well. 
It represents the available images for a given version so it does not belong to `.ci` only  🤷  (for example: we could use it in our snapshot/stable releases workflows to update the README with the avaialble images).

Reply:
I guess we will figure with the usage ;)